state/protocol/badger/state.go

```diff
@@ -46,6 +46,8 @@ type State struct {
 	// larger than the height of the root block of the spork, (also cached below as
 	// `sporkRootBlockHeight`), for instance if the node joined in an epoch after the last spork.
 	rootHeight uint64
```

Comment:
```suggestion
	finalizedRootHeight uint64
```
update this name for consistency with elsewhere, and disambiguation from `sealedRootHeight`